### PR TITLE
created all classes

### PR DESCRIPTION
**Pull request by YuraBondarenko**

### CODE REVIEW

File: src/main/java/core/basesyntax/Drawable.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Drawable {
+    void drawingFigures();
```
Comment (RomanDubovskyi) [inline src/main/java/core/basesyntax/Drawable.java:4]:
```suggestion
    void drawFigure();
```

File: src/main/java/core/basesyntax/Colors.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum Colors {
```
Comment (RomanDubovskyi) [inline src/main/java/core/basesyntax/Colors.java:3]:
```suggestion
public enum Color {
```

File: src/main/java/core/basesyntax/FigureProducer.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureProducer {
+    public static final int figure = 5;
```
Comment (RomanDubovskyi) [inline src/main/java/core/basesyntax/FigureProducer.java:6]:
```suggestion
    public static final int FIGURE_COUNT = 5;
```
we declare constant in upper case
do we use this constant only in this class, if yes let's make private

File: src/main/java/core/basesyntax/FigureProducer.java [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+    public static Figure getFigure() {
```
Comment (RomanDubovskyi) [inline src/main/java/core/basesyntax/FigureProducer.java:8]: why methods are static in this class?

**Review by RomanDubovskyi — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/FigureProducer.java (hunk 4 above)
Comment (YuraBondarenko) [reply src/main/java/core/basesyntax/FigureProducer.java:8]: to access this method in the class

Comment (RomanDubovskyi) [reply src/main/java/core/basesyntax/FigureProducer.java:8]: you can access not only static methods 
FigureProducer figureProducer = new FigureProducer();
figureProducer.getFigure() 
will work in main mehtod

File: src/main/java/core/basesyntax/Circle.java [hunk 5]
```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
```
Comment (RomanDubovskyi) [inline src/main/java/core/basesyntax/Circle.java:3]: add getRadius() method, so we can get this value from object

File: src/main/java/core/basesyntax/FigureProducer.java [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+                return new IsoscelesTrapezoid(getColor(), getRandomNumber(),
+                        getRandomNumber(), getRandomNumber());
+        }
+    }
+
+    public static int getRandomNumber() {
+        return new Random().nextInt(10) + 1;
+    }
+
+    public static Color getColor() {
```
Comment (RomanDubovskyi) [inline src/main/java/core/basesyntax/FigureProducer.java:26]: if we use this method only inside of this class make it private 
```suggestion
    public Color getColor() {
```

File: src/main/java/core/basesyntax/FigureProducer.java [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+            case 3: return new Square(getColor(), getRandomNumber());
+            case 4: return new RightTriangle(getColor(), getRandomNumber(), getRandomNumber());
+            case 5:
+            default:
+                return new IsoscelesTrapezoid(getColor(), getRandomNumber(),
+                        getRandomNumber(), getRandomNumber());
+        }
+    }
+
+    public static int getRandomNumber() {
```
Comment (RomanDubovskyi) [inline src/main/java/core/basesyntax/FigureProducer.java:22]: same here

File: src/main/java/core/basesyntax/FigureProducer.java [hunk 8]
```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureProducer {
+    private static final int FIGURE_COUNT = 5;
+
+    public static Figure getFigure() {
```
Comment (RomanDubovskyi) [inline src/main/java/core/basesyntax/FigureProducer.java:8]:
```suggestion
    public Figure getFigure() {
```

**Review by RomanDubovskyi — CHANGES_REQUESTED** [review]

**Review by RomanDubovskyi — APPROVED** [review]

**Pull request closed without merging**